src/servers/cfworker.ts

```diff
@@ -53,7 +53,8 @@ export class CloudflareWorkerServer extends Server {
                   }
                 })
               );
-          }
+          },
+          { cloudflare: true, ...env }
```

Comment:
```suggestion
          { server: 'cloudflare', ...env }
```

This would be better so other contexts can use the same prop.